mappers/bluetooth_mapper/watcher/watcher.go

```diff
@@ -137,54 +145,52 @@ func (w *Watcher) EquateTwinValue(deviceID string) error {
 	helper.Wg.Wait()
 	twinUpdated := false
 	for _, twinAttribute := range w.DeviceTwinAttributes {
-		if helper.TwinResult.Twin[twinAttribute.Name] != nil {
-			if helper.TwinResult.Twin[twinAttribute.Name].Expected != nil && ((helper.TwinResult.Twin[twinAttribute.Name].Actual == nil) && helper.TwinResult.Twin[twinAttribute.Name].Expected != nil || (*helper.TwinResult.Twin[twinAttribute.Name].Expected.Value != *helper.TwinResult.Twin[twinAttribute.Name].Actual.Value)) {
-				klog.Infof("%s Expected Value : %s", twinAttribute.Name, *helper.TwinResult.Twin[twinAttribute.Name].Expected.Value)
-				if helper.TwinResult.Twin[twinAttribute.Name].Actual == nil {
-					klog.Infof("%s  Actual Value: %v", twinAttribute.Name, helper.TwinResult.Twin[twinAttribute.Name].Actual)
-				} else {
-					klog.Infof("%s Actual Value: %s", twinAttribute.Name, *helper.TwinResult.Twin[twinAttribute.Name].Actual.Value)
+		syncedTwinAttribute := helper.TwinResult.Twin[twinAttribute.Name]
+		if syncedTwinAttribute == nil {
+			return errors.New("The attribute: " + twinAttribute.Name + " does not exist for this device")
+		}
+		if syncedTwinAttribute.IsSynced() {
+			continue
+		}
+		klog.Infof("%s Expected Value : %s", twinAttribute.Name, *syncedTwinAttribute.Expected.Value)
+		if syncedTwinAttribute.Actual == nil {
+			klog.Infof("%s  Actual Value: %v", twinAttribute.Name, syncedTwinAttribute.Actual)
```

Comment:
Is it for debug? We shall use below log rather than use a if block:
```
klog.Infof("%s Actual Value: %v", twinAttribute.Name, syncedTwinAttribute.Actual)
```

Reply:
Also, use lower log level like `klog.V(2).Infof` is a good choice for debug log.

Reply:
thanks, will do that